location/build.gradle

```diff
@@ -0,0 +1,39 @@
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+}
+
+android {
+    compileSdkVersion rootProject.androidCompileSdk
+
+    defaultConfig {
+        minSdkVersion rootProject.androidMinSdk
+        targetSdkVersion rootProject.androidTargetSdk
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles "consumer-rules.pro"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+}
+
+dependencies {
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+    implementation 'androidx.core:core-ktx:1.6.0'
```

Comment:
I've started noticing versions drifting apart. Should we maybe always have a version constant for dependencies that show up in more than one module?

Reply:
This is an interesting one because although it doesn't feel quite right, I'm not sure if it really matters unless we see a conflict at compile time. Maybe we should add an investigation issue to look into dependency versions in multimodule setups?

Reply:
Investigation issue works for me! In general I don't expect mismatched versions would generally cause a problem but there might be occasional subtle issues that drive us crazy.

I spun my wheels for a while when I upgraded the AndroidX fragment version. It turned out to need to be upgraded across three modules and the error states before I figured that out were really hard to understand.

Reply:
Done: https://github.com/getodk/collect/issues/4691